### PR TITLE
Arrowhead-class 

### PR DESCRIPTION
**Pull request by TheNeoGamer42**
## About The Pull Request
adds ship, vaguely submarine feeling edition, powered only on RTGs, and if you put a pacman on it, I will come to your house, uninstall byond, and unplug your computer.

## Why It's Good For The Game
ship :)
arrowhead
## Changelog
:cl:
add: The Arrowhead-class Long Range Scoutship exists now.
/:cl:

### CODE REVIEW

**Comment by Dimasw99** [comment]
pics or it never happened

**Comment by TheNeoGamer42** [comment]
Image added.

**Comment by Kurgis** [comment]
Could use a space heater in Atmospherics due to the main airlock exchanging planet atmos with the ship, usually heating or cooling it in unwanted ways.

File: _maps/shuttles/shiptest/independent_arrowhead.dmm [hunk 1]
```diff
@@ -0,0 +1,3188 @@
...
+	pixel_y = 13
+	},
+/turf/open/floor/plasteel/showroomfloor,
+/area/ship/crew)
+"Or" = (
+/obj/machinery/computer/autopilot{
+	dir = 8
+	},
+/turf/open/floor/plasteel/tech,
+/area/ship/bridge)
```
Comment (Vexylius) [inline _maps/shuttles/shiptest/independent_arrowhead.dmm]: Autopilot console was removed from the game, can you please remove it from this ship?

**Review by Vexylius — CHANGES_REQUESTED** [review]

**Comment by Apogee-dev** [comment]
please make sure the dirs for your wallmounts (buttons, lightswitches, fire alarms, intercoms, etc) are all correct for the wall face they're on. also, this isn't a hard requirement but you may want to add regular intercoms to the ship as they're pretty standard on mid-size ships

File: _maps/shuttles/shiptest/independent_arrowhead.dmm (hunk 1 above)
Comment (Fikou) [reply _maps/shuttles/shiptest/independent_arrowhead.dmm]:
```suggestion
"Or" = (
/obj/structure/frame/computer{
	dir = 8
	},
/turf/open/floor/plasteel/tech,
/area/ship/bridge)
```

**Review by Vexylius — APPROVED** [review]

**Comment by TheNeoGamer42** [comment]
Why was this merged? I wasn't done and I haven't had time to bother updating it properly. It literally still has the old autopilot console that was removed.

**Comment by TheNeoGamer42** [comment]
There's also quite a few other issues like I haven't gotten around to putting the proper medhud in, or the right medkits, or the right defib either. The medkits are empty, the medhud is the cyborg one, and the defib has no battery.

**Comment by ZephyrTFA** [comment]
If you do not want your PR to be merged you need to mark it as a draft

**Comment by TheNeoGamer42** [comment]
It was marked stale and autoclosed. And then reopened for some reason. Not my fault that someone opened it again after it had autoclosed when there was issues. I even mentioned to vexy some time ago I needed to work on this and they merged it anyways?

**Comment by Fikou** [comment]
> It literally still has the old autopilot console that was removed.

it doesnt

> There's also quite a few other issues like I haven't gotten around to putting the proper medhud in, or the right medkits, or the right defib either. The medkits are empty, the medhud is the cyborg one, and the defib has no battery.

you can change this whenever

**Comment by triplezeta** [comment]
kind of based tbh

**Comment by TheNeoGamer42** [comment]
ah yes, because adding an incomplete map with known issues is based apparently, and also last I checked it still did have the autopilot unless someone's been changing it that isn't me

**Comment by triplezeta** [comment]
if you checked the commit history you would know that is the case yes

**Comment by TheNeoGamer42** [comment]
Yeah, I see that commit now, but still, I don't see why this was merged at all still.

**Pull request merged**